src/styles/variables.ts

```diff
@@ -69,7 +69,9 @@ export default {
     fontSizeSignInHeroMedium: 38,
     fontSizeSignInHeroXSmall: 26,
     fontSizeSignInHeroSmall: 28,
+    fontSizeSignInHeroSmallLineHeight: 30.8,
     fontSizeSignInHeroBody: 20,
+    lineHeightMultiplier: 1.1,
```

Comment:
Why do we need a multiplier? Curious what this does exactly.

Reply:
I'm not sure why this was implemented this way, I've unified it with how we handled `textHero`, which means I've added this:

```
textHeroSmall: {
        fontSize: variables.fontSizeHeroSmall,
        fontFamily: FontUtils.fontFamily.platform.EXP_NEW_KANSAS_MEDIUM,
        lineHeight: variables.lineHeightHeroSmall
},
```

And variables needed (with line height being pre-calculated to be 125% of font size):

```
fontSizeHeroSmall: 28,
lineHeightHeroSmall: 35,
```

This is the exact way we currently have `textHero` implemented and should make it all more readable.

Reply:
Cool. See my comment above but I am questioning why we even need a special style for "textHeroSmall" when we have an h1 style we can reuse here. Maybe @dubielzyk-expensify has more context though.